### ISSUE TITLE
Literal suffixes should be upper case

### ISSUE THREAD

**Issue by tvsonar** [SonarSource/sonarlint-vs#366]
```
var x = 10l; //Noncompliant, is it 101 or 10? It's hard to read.
var y = 10L;
```

R# ID: LongLiteralEndingLowerL

https://jira.sonarsource.com/browse/RSPEC-818

**Comment by tvsonar** [comment]
The case of `l` vs `L` is handled by CS0078. But that is not triggered in for the other suffixes

**Comment by tvsonar** [comment]
There are too many issues reported by this rule. I would probably disable it by default. What do you thing @dbolkensteyn?

**Comment by tvsonar** [comment]
I got rid of 80% of the issues by properly filtering for literal suffixes and not for `0xfff`, or `12e-34` like literals.

**Comment by OsirisTerje** [comment]
This rule reacts to floating point suffixes too, like 1.0f.   
I can't see any reason that F is better than f here.
